README.md

```diff
@@ -171,7 +171,7 @@ docker run --name gitlab-postgresql -d \
     --env 'DB_USER=gitlab' --env 'DB_PASS=password' \
     --env 'DB_EXTENSION=pg_trgm' \
     --volume /srv/docker/gitlab/postgresql:/var/lib/postgresql \
-    sameersbn/postgresql:10-2
+    sameersbn/postgresql:12-20200524
```

Comment:
Same

Reply:
didn't saw that it is still unsupported, had only read PostgreSQL 11+ in the requirements. Updated README.md